src/pause/138CC0.c

```diff
@@ -1,15 +1,145 @@
 #include "common.h"
 
+// Probably only used here, but could theoretically be used in the main menu too
+typedef struct {
+    s32 cursorX;
+    s32 cursorY;
+    s32 baseMsgID;
+} StatsEntryData; // size = 0xC
+
+// Probably .data or .bss in this file, not used anywhere else
```

Comment:
These are data symbols, which you can tell since they were renamed in a .data.s file as well. I don't think we need the comment. We comment bss when it's not handled yet, but this data will be migrated and it works as-defined